restconnector/RequestHandler.cpp

```diff
@@ -41,27 +43,33 @@ void RequestHandler::operator() ( const server::request &request, server::respon
     server::string_type ip = source( request );
     RestZeqTranslator restZeqTranslator;
 
-    if ( restZeqTranslator.findCommand( request.destination ) )
+    if ( !restZeqTranslator.isASupportedCommand( request.destination ) )
     {
-        std::string method = request.method;
-
-        ::zeq::Event zeqEvent = restZeqTranslator.translate();
-
-        if( method == "PUT" )
-            processPUT_( zeqEvent );
-        else if( method == "GET" )
-            processGET_( zeqEvent );
-        else if( method == "POST" )
-            processPOST_( zeqEvent );
-
-        response = response_;
+        response = server::response::stock_reply( server::response::bad_request,
+                                                  "Command not found." );
     }
     else
     {
-        response = server::response::stock_reply( server::response::bad_request,
-                                                  "Command not found." );
+        std::string method = request.method;
```

Comment:
why copy the string? const ref at least? It's just a shortcut from what I can see